modules/pensions/app/models/pensions/saved_claim.rb

```diff
@@ -69,52 +69,12 @@ def email
     end
 
     ##
-    # enqueue the sending of the submission confirmation email
+    # utility function to retrieve claimant first name from form
     #
-    # @see VANotify::EmailJob
+    # @return [String] the claimant first name
     #
-    def send_confirmation_email
```

Comment:
Yeah this move made sense to do in this ticket

Reply:
yeah, was here or in a separate one specifically for getting the pension/burial emails rigged up